airflow/utils/configuration.py

```diff
@@ -21,6 +21,7 @@
 from tempfile import mkstemp
 
 from airflow.configuration import conf
+from airflow.platform import IS_WINDOWS
```

Comment:
should it be "from airflow.utils.platform import IS_WINDOWS" ?

Reply:
True, thanks for pointing this out